docs-source/content/samples/simple/tanzu-kubernetes-service/_index.md

```diff
@@ -0,0 +1,759 @@
+---
+title: "Tanzu Kubernetes Service"
+date: 2020-11-16T13:34:31-05:00
+weight: 9
+description: "Sample for using the operator to set up a WLS cluster on the Tanzu Kubernetes Service."
+---
+
+
+This sample demonstrates how to use the Oracle [WebLogic Server Kubernetes Operator](/weblogic-kubernetes-operator/) (hereafter “the operator”) to set up a WebLogic Server (WLS) cluster on the Tanzu Kubernetes Grid (TKG).
+After performing the sample steps,, your WLS domain runs on a TKG Kubernetes cluster instance and you can manage your WLS domain by using the WebLogic Server Administration Console.
+
+TKG is a managed Kubernetes Service that lets you quickly deploy and manage Kubernetes clusters. To learn more, see the [Tanzu Kubernetes Grid (TKG)](https://docs.vmware.com/en/VMware-Tanzu-Kubernetes-Grid/1.2/vmware-tanzu-kubernetes-grid-12/GUID-index.html) overview page.
+
+#### Contents
+
+ - [Prerequisites](#prerequisites)
+   - [Create the Tanzu kubernetes cluster](#create-the-tanzu-cluster)
+   - [Oracle container registry](#oracle-container-registry)
+ - [Install WebLogic Server Kubernetes Operator](#install-weblogic-server-kubernetes-operator)
+ - [Create docker image](#create-docker-image)
+ - [Create WebLogic domain](#create-weblogic-domain)
+ - [Invoke the web application](#invoke-the-web-application)
+
+#### Prerequisites
+
+This sample assumes the following prerequisite environment setup.
+
+* WebLogic Server Kubernetes operator: This document was tested with version v3.1.0.
+* Operating System: GNU/Linux.
+* [Git](https://git-scm.com/downloads); use `git --version` to test if `git` works.  This document was tested with version 2.17.1.
+* TKG CLI; use `tkg version` to test if tkg works. This document was tested with version v1.1.3.
+* [kubectl](https://kubernetes-io-vnext-staging.netlify.com/docs/tasks/tools/install-kubectl/); use `kubectl version` to test if `kubectl` works.  This document was tested with version v1.18.6.
+* [helm](https://helm.sh/docs/intro/install/); version 3.1 or later; use `helm version` to check the `helm` version.  This document was tested with version v3.2.1.
```

Comment:
(https://helm.sh/docs/intro/install/)**;** -> don't need the semi-colon here; only after the word later.

Reply:
fixed